azkaban-web-server/src/web/js/azkaban/util/flow-loader.js

```diff
@@ -81,49 +81,65 @@ var nodeClickCallback = function (event, model, node) {
     var flowRequestURL = contextURL + "/manager?project=" + projectName
         + "&flow=" + node.flowId;
     if (node.expanded) {
-      menu = [{
-        title: "Collapse Flow...", callback: function () {
-          model.trigger("collapseFlow", node);
+      menu = [
+        {
+          title: "Collapse Flow...", callback: function () {
+            model.trigger("collapseFlow", node);
+          }
+        },
+        {
+          title: "Collapse all Flows...", callback: function () {
```

Comment:
Consider changing from lower case to upper case: "Collapse All Flows..."?

Reply:
sure, I will change it.

Reply:
Please change all the places to keep consistency.